### PR TITLE
Nm 104 local storage

### PR DESCRIPTION
**Pull request by marek-palenik**

### CODE REVIEW

File: src/lib/localStorageUtils.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+export const saveToStorage = (item: string, name: string) => {
+
+  try {
+    localStorage.setItem(item, name);
+  } catch (e) {
+    console.error(e);
+  }
+};
+
+export const loadFromStorage = (itemName: string): any => {
```
Comment (dullus) [inline src/lib/localStorageUtils.ts:11]: return type will be `: string | undefined`

File: src/lib/localStorageUtils.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+
+export const saveToStorage = (item: string, name: string) => {
+
+  try {
+    localStorage.setItem(item, name);
+  } catch (e) {
+    console.error(e);
```
Comment (dullus) [inline src/lib/localStorageUtils.ts:7]: use debug function from utils.ts instead of console.error

File: src/lib/localStorageUtils.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+export const loadFromStorage = (itemName: string): any => {
+
+  try {
+    const item = localStorage.getItem(itemName);
+    if (item === null) {
+      return undefined;
+    }
+    return item;
+  } catch (e) {
+    console.error(e);
```
Comment (dullus) [inline src/lib/localStorageUtils.ts:20]: use debug function from utils.ts instead of console.error

File: src/lib/localStorageUtils.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    return undefined;
+  }
+};
+
+export const removeFromStorage = (itemName: string): void => {
+
+  try {
+    localStorage.removeItem(itemName);
+  } catch (e) {
+    console.error(e);
```
Comment (dullus) [inline src/lib/localStorageUtils.ts:30]: use debug function from utils.ts instead of console.error

File: src/lib/localStorageUtils.ts (hunk 3 above)
Comment (marek-palenik) [reply src/lib/localStorageUtils.ts]: hopefully it is fixed, as i did not find any reference on debug from utils.tsx

**Pull request merged**